WordPress/Classes/ViewRelated/Post/PostListFilter.h

```diff
@@ -10,11 +10,12 @@ typedef NS_ENUM(NSUInteger, PostListStatusFilter) {
 @interface PostListFilter : NSObject
 
 @property (nonatomic, assign) BOOL hasMore;
+@property (nonatomic, strong) NSDate *oldestPostDate;
 @property (nonatomic, assign) PostListStatusFilter filterType;
 @property (nonatomic, strong) NSString *title;
 @property (nonatomic, strong) NSArray *statuses;
-@property (nonatomic, strong) NSPredicate *predicateForFetchRequest;
 
 + (NSArray *)newPostListFilters;
+- (NSPredicate *)predicateForFetchRequest;
```

Comment:
I'm not sure I understand this change.  It looks like it might be to make `predicateForFetchRequest` read-only. If so, maybe just add the `readonly` modifier and re-declare the property `readwrite` in the implementation?